lib/api/v3/utilities/path_helper.rb

```diff
@@ -66,6 +66,10 @@ def self.category(id)
             "#{root}/categories/#{id}"
           end
 
+          def self.create_work_package_form(project_id)
+            "#{work_packages_by_project(project_id)}/form"
+          end
```

Comment:
For the record: You might want to add specs for the path helper. (We discussed and you already have that on your change list)